### PR TITLE
implement both suggestions from issue #548

### LINKED ISSUE #548: Digraph for `Ṫ0p`
Remove the last, prepend a zero.

This is very useful in anything involving prefixes. It'd also be nice to have one for `¦Ṫ0p` - ^ but with cumulative sums.

### PR DESCRIPTION
**Pull request by Lyxal**
closes #548

### CODE REVIEW

File: vyxal/elements.py [hunk 1]
```diff
@@ -3097,6 +3105,18 @@ def run_length_decoding(lhs, ctx):
         return LazyList(temp)
 
 
+def sans_last_prepend_zero(lhs, ctx):
```
Comment (cgccuser) [inline vyxal/elements.py:3108]: Could we rename this `drop_last_prepend_zero` instead? `drop` feels more intuitive than `sans`, even if `sans` is fancier.

File: vyxal/elements.py [hunk 2]
```diff
@@ -3097,6 +3105,18 @@ def run_length_decoding(lhs, ctx):
...
+    """Element Þr
+    Remove the last item of a list and prepend 0
+    """
+    ts = vy_type(lhs)
+    return {
+        NUMBER_TYPE: lambda: tail_remove(lhs, ctx),  # prepending a 0 to
```
Comment (cgccuser) [inline vyxal/elements.py:3114]: Hmm, maybe have a more useful overload for numbers then?

**Review by cgccuser — APPROVED** [review]
LGTM overall

**Review by pxeger — APPROVED** [review]

**Comment by pxeger** [comment]
I feel like this would be better to implement with a single-byte overload which prepends zero

**Review by PyGamer0 — APPROVED** [review]

**Comment by Lyxal** [comment]
> I feel like this would be better to implement with a single-byte overload which prepends zero

Then you have to have an element that has number and string overloads that can get away with not being vectorised by default.

File: vyxal/elements.py (hunk 1 above)
Comment (chunkybanana) [reply vyxal/elements.py:3119]: ^

**Pull request merged**